packages/server/controllers/guilds.js

```diff
@@ -60,7 +61,65 @@ async function guildExists(guildId) {
   return !!guild;
 }
 
-async function getGuildMembers(guildId) {
+async function addGuildMember(guildId, userId) {
+  return await prisma.guildMembers.create({
+    data: {
+      userId: userId,
+      guildId: guildId,
+      points: 0,
+      role: GuildMemberRole.Member,
+    },
+  });
+}
+
+// Returns a record containing the userId, guildId, points, role, firstName, lastName, avatar
+async function getGuildMember(guildId, userId) {
+  const getMember = await prisma.guildMembers.findUnique({
+    where: {
+      guildId_userId: {
+        guildId: guildId,
+        userId: userId,
+      },
+    },
+    include: {
+      members: {
+        select: {
+          firstName: true,
+          lastName: true,
+          avatar: true,
+        },
+      },
+    },
+  });
+
+  if (getMember) {
+    // temp for functionality, replace later
+    const flattenedData = Object.entries(getMember)
+      .flatMap(([key, value]) => {
+        if (
+          typeof value === "object" &&
+          value !== null &&
+          !Array.isArray(value)
+        ) {
+          return Object.entries(value).map(([subKey, subValue]) => [
+            subKey,
+            subValue,
+          ]);
+        } else {
+          return [[key, value]];
+        }
+      })
+      .reduce((acc, [key, value]) => {
+        acc[key] = value;
+        return acc;
+      }, {});
+
+    return flattenedData;
+  }
```

Comment:
Remember to replace this with a better data flattener since you say this is temporary for now.